.tool-versions

```diff
@@ -1,4 +1,4 @@
-golang     1.20.14
+golang     1.22.0
```

Comment:
That's potentially too restrictive since we're providing a library. The Go toolchain typically provides support for ~2 major versions, and a bunch of OSS libraries do the same.

In principle, we could split the code into different modules so that we can aggressively bump the version for the CLI and leave the version bound for the bindings lower, but that would make things more complicated.